static/js/components/EducationTab.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Button from 'react-mdl/lib/Button';
+import ProfileTab from "../util/ProfileTab";
+import { boundEducationSelectField,
+  boundEducationMonthField, boundEducationYearField,
+  saveAndContinue } from '../util/profile_edit';
+import {
+  toggleEducationLevel,
+  openEducationForm,
+  closeEducationForm
+} from '../actions';
+import { Grid, Cell, Switch, Dialog, DialogTitle, DialogContent, DialogActions, FABButton, Icon } from 'react-mdl';
+
+class EducationTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.boundEducationMonthField = boundEducationMonthField.bind(this);
+    this.boundEducationYearField = boundEducationYearField.bind(this);
+    this.educationLevelOptions = [
+      { value: 'hs', label: "High school", enabled: true },
+      { value: 'a', label: 'Associate degree', enabled: true },
+      { value: 'b', label: "Bachelor's degree", enabled: true },
+      { value: 'm', label: "Master's or professional degree", enabled: false },
+      { value: 'p', label: "Doctorate", enabled: false }
+    ];
+    this.educationLevelLabels = {
```

Comment:
Can you create this object using the data from `educationLevelOptions` to reduce duplication?